nb/documentation/interactions_between_geometry/interactions_between_geometry.nb

```diff
@@ -82,6 +97,15 @@ Performs the hull operation on the input shapes.`;
 
 Hull(Arc(10), Box(5).move(12, 12, 0), Arc(20).x(40)).view();
 
+md`
+---
+### Mask
+Mask is used to create a void around something. Mostly used for adding play.
```

Comment:
Rather, a mask is used instead of the masked thing for cutting.

e.g., you might want to mask a bolt thread with a simple cylinder to cut a bolt hole rather than threads through interpenetrated shapes.

Usually it makes sense to use a void for the mask.

Reply:
I'm not sure I understand the difference. In both cases we want to have something cut a larger hole than it's own size, right?

Reply:
Sure, I guess the point is that the relationship between mask and void is accidental rather than fundamental.

Reply:
I see, that is important. It's just taking advantage of the .void feature rather than being an alternative to void.